bin/gem2rpm

```diff
@@ -25,17 +30,27 @@ if options[:print_template_file]
 end
 
 if rest.size != 1
-  $stderr.puts "Missing GEMFILE\n\n"
-  $stderr.puts opts
+  show_err('Missing GEMFILE', options)
   exit(1)
 end
 gemfile = rest[0]
+out_dir =
+  if options[:directory]
```

Comment:
Could you please extract it into helper method? Probably somewhere into lib/gem2rpm.rb